pyfr/solvers/base/system.py

```diff
@@ -111,7 +113,8 @@ def _load_eles(self, rallocs, mesh, initsoln, nregs, nonce):
             k = f'spt_{etype}_p{rallocs.prank}'
 
             try:
```

Comment:
I have a question here, (sorry if I'm mistaken I think we have already talked abot this) but it seems that the computation of `linoff` supposes that the elements are ordered based on if they are `linear` or not. However, this ordering is only imposed when partitioning the mesh object. So if no partition is done, all elements will be treated as `curved` isn't it?